test/e2e/data/infrastructure-nutanix/v1.2.4/base/cluster-with-kcp.yaml

```diff
@@ -81,7 +81,7 @@ spec:
           spec:
             containers:
               - name: kube-vip
-                image: ghcr.io/kube-vip/kube-vip:v0.6.4
+                image: ghcr.io/kube-vip/kube-vip:v0.7.0
```

Comment:
IMO, we should leave the versioned templates as is so they reflect the status of the template as of that release.